docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/webhook-disable.md

```diff
@@ -0,0 +1,425 @@
+       projectUid: XXXXXX
+     imageRedirectWebhook:
+       enable: false
+   ```
+
+5. Follow the rest of the [Build Edge Artifact](../../edgeforge-workflow/palette-canvos/palette-canvos.md) guide and
+   build the Installer ISO with the user data configurations. The Edge clusters provisioned with the ISO will no longer
+   automatically redirect image pull requests to the external registry or the local Harbor registry.
+
+### Redirect Image Pull
+
+The process to redirect image pulls varies by Kubernetes distribution as well as your registry setup. This section
+provides an example that shows how you might customize the image pull behavior of your Edge cluster using PXK-E.
+
+6. Log in to [Palette](https://console.spectrocloud.com).
+
+7. From the left **Main Menu**, click **Profiles**. Click on the profile you use to deploy your Edge cluster.
+
+8. (PXK-E Only) In the Kubernetes layer of the profile, include the following lines in the `initramfs` stage to adjust
+   the containerd configuration to supports reading additional files, which you will use to configure the redirect
+   behavior and provide registry credentials.
+
+   ```yaml
+   stages:
+     initramfs:
+       - name: "Manage containerd config"
+         files:
+           - path: /etc/containerd/config.toml
+             permissions: 0644
+             owner: 0
+             group: 0
+             content: |-
+               version = 2
+               imports = ["/etc/containerd/conf.d/*.toml"]
+               [plugins]
+                 [plugins."io.containerd.grpc.v1.cri"]
+                   sandbox_image = "registry.k8s.io/pause:3.9"
+                   enable_unprivileged_ports = true
+                   enable_unprivileged_icmp = true
+                 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+                   runtime_type = "io.containerd.runc.v2"
+                 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+                   SystemdCgroup = true
+                 [plugins."io.containerd.grpc.v1.cri".registry]
+                   config_path = "/etc/containerd/certs.d"
+   ```
+
+   :::info
+
+   This configuration change is only needed to PXK-E. Since it changes the `initramfs` stage, this will require a reboot
+   of the node. If you are using K3s or RKE2, the ability fo read additional files is enabled by default and you don't
+   need to add this configuration.
+
+   :::
+
+9. In the Kubernetes layer of the profile, include the following lines in the reconcile stage. For more information,
+   refer to [Cloud-init Stages](../../edge-configuration/cloud-init.md).
+
+   ```yaml
+   stages:
+    reconcile:
+        - name: "Redirect registries"
+            - path: /etc/containerd/certs.d/gcr.io/hosts.toml
+              owner: 0
+              group: 0
+              permissions: 0644
+              content: |-
+                server = "https://gcr.io"
+                [host."https://gcr-io-mirror.company.local"]
```

Comment:
Should we highlight this line? My guess is this is where the user would specify their registry. It would help the reader if we call this out 😄